app/javascript/react/src/components/CelestialTile.js

```diff
@@ -1,16 +1,18 @@
 import React, {Component} from 'react';
+import { Link } from 'react-router'
 
-const Celestial = (props) => {
+
+const CelestialTile = (props) => {
   return(
     <div>
       <div className="index-image">IMAGE</div>
       <div className="index-details">
-        <h1 className="celestial-name">{props.name}</h1>
+        <Link to={`/celestials/${props.id}`}><h1 className="celestial-name">{props.name}</h1></Link>
         <p>{props.type}</p>
```

Comment:
Since line 10 is a little long I'd recommend adding line breaks before the h1 tag and after the closing h1 tag.